lib/engine/step/base.rb

```diff
@@ -67,7 +67,9 @@ def active_entities
       end
 
       def round_state
-        {}
+        {
+          trains_bought: [],
```

Comment:
this should not go in the base step, only in the train buying step

Reply:
Fixed